src/limitedmap.h

```diff
@@ -66,8 +66,7 @@ class limitedmap
     }
     void update(const_iterator itIn, const mapped_type& v)
     {
-        // TODO: When we switch to C++11, use map.erase(itIn, itIn) to get the non-const iterator.
-        iterator itTarget = map.find(itIn->first);
+        iterator itTarget = map.erase(itIn, itIn);
```

Comment:
@laanwj this trick is explained here: https://stackoverflow.com/questions/765148/how-to-remove-constness-of-const-iterator,  and appears to be pretty common.